src/Message/Formatter.php

```diff
@@ -141,7 +141,19 @@ private function defaultFormat(Message $message, array $commonContext): string
     private function getTime(Message $message): string
     {
         $timestamp = (string) $message->context('time', microtime(true));
-        $format = strpos($timestamp, '.') === false ? 'U' : 'U.u';
+
+        switch (true) {
```

Comment:
What's the benefit of using `switch(true)`?

Reply:
fixed.

Reply:
It's actually not.

Reply:
See https://github.com/yiisoft/log/pull/77/files#r837271425